src/main/java/dev/bernasss12/bebooks/client/gui/ModConfig.java

```diff
@@ -97,7 +99,7 @@ public static void loadEnchantmentData() {
 
     public static void saveEnchantmentData() {
         File file = new File(FabricLoader.getInstance().getConfigDir().toFile(), "bebooks/enchantment_data.json");
-        Gson gson = new Gson();
+        Gson gson = new GsonBuilder().setPrettyPrinting().disableHtmlEscaping().create();
```

Comment:
Pretty printing! Also, just in case it, HTML escaping should always be disabled (Google, pls!)